tensorboard/plugins/graph/tf_graph_controls/tf-graph-controls.html

```diff
@@ -412,24 +426,16 @@
       <br style="clear: both">
       <div>Devices included in stats:</div>
       <div class="deviceList">
-        <table>
-        <template is="dom-repeat" items="[[_getDevices(devicesForStats)]]">
-          <tr>
-            <td>
-              <input type="checkbox" value$="[[item.device]]" checked$="[[item.used]]" on-click="_deviceCheckboxClicked"/>
-            </td>
-            <td>
-              <div>
-                <span>[[item.suffix]]</span>
-                <template is="dom-if" if="[[item.ignoredMsg]]">
-                  <paper-icon-button icon="help" class="help-icon"></paper-icon-button>
-                  <paper-tooltip position="right" animation-delay="0">[[item.ignoredMsg]]</paper-tooltip>
-                </template>
-              </div>
-            </td>
-          </tr>
+        <template is="dom-repeat" items="[[_currentDevices]]">
+          <div id="devices-checkbox" class="color-legend-row">
+            <span><input style="text-align:left;vertical-align:middle" type="checkbox" value$="[[item.device]]" checked$="[[item.used]]" on-click="_deviceCheckboxClicked"/></span>
```

Comment:
Since you defined the `#devices-checkbox` CSS block above, I don't believe you need this inline style anymore.

Reply:
I would think so too.  Here are two screen shots (with the inline style and without it)

With it:
![image](https://user-images.githubusercontent.com/35349680/35350627-5ce0f5d0-00f3-11e8-958b-c913386ea135.png)

Without it:
![image](https://user-images.githubusercontent.com/35349680/35350642-657eae12-00f3-11e8-8f46-78e64e404d76.png)

The style parameters are telling the checkbox to be centered vertically and left within the encompassing SPAN.  Without it, the checkbox is at the top of its parent SPAN (thus not aligned with the label text to its right).

Reply:
Got it - thanks! Could we move this inline CSS into an embedded one? ie,

```css
.devices-checkbox input {
  text-align:left;
  vertical-align:middle;
}
```

Reply:
Nice.  Done.